lib/workers/repository/init/merge.ts

```diff
@@ -212,7 +212,13 @@ export async function mergeRenovateConfig(
   }
   // Decrypt after resolving in case the preset contains npm authentication instead
   let resolvedConfig = await decryptConfig(
-    await presets.resolveConfigPresets(decryptedConfig, config),
+    await presets.resolveConfigPresets(
+      decryptedConfig,
+      config,
+      [],
+      [],
+      'on' // shallow log config
```

Comment:
What goes wrong if shallow log is off?

Reply:
When its off then the logic for logging shallow config will be entirely skipped (the diffs in this pr)